pages/guides/how-to-schedule-price-updates-with-gelato.mdx

```diff
@@ -0,0 +1,125 @@
+various parameters such as the price feeds to update, the update frequency,
+price deviation threshold and can be dynamically updated at any time.
+
+## Steps
+
+#### 1. Join Gelato Web3 Functions whitelist
+
+Web3 Functions are currently in private beta.
+Get whitelisted in the Web3 Function private Beta. If you are not, reach out to
+the Gelato team [here](https://form.typeform.com/to/RrEiARiI).
+
+#### 2. Configure your task
+
+The Gelato task reads a yaml configuration file from a GitHub gist.
+Create a GitHub gist and add a file called `config.yaml`.
+The `config.yaml` file should follow the example `config.yaml` files from this
+
+[//]: # "TODO: redirect this to pyth fork of repo)"
+
+[directory](https://github.com/gelatodigital/w3f-pyth-poc-v2/tree/master/web3-functions/pyth-oracle-w3f-priceIds)
+[example gist](https://gist.github.com/swimricky/18b2a5ad9c1a605f1cf5c19ac1d2f1d9)
+
+Once the task is running, you can update this configuration to add more price feeds or change the update parameters.
+The task will automatically pick up configuration changes.
+
+#### 3. Create the task
+
+Use the link below to auto-populate the task parameters with the Pyth web3 function code:
+
+[//]: # "TODO: check/update this link"
+
+https://beta.app.gelato.network/new-task?cid=QmTFQHpiThqkVKvuW2KU8cETNBdyXHB9uP34woL6Vc5w6g
+
+The pyth web3 function code is deployed to IPFS, so you can use it via the cid/link above.
+you can find a copy of the web3 function code [here](https://github.com/gelatodigital/w3f-pyth-poc-v2/tree/master/web3-functions/pyth-oracle-w3f-priceIds)
+
+Choose network, and then in the "Task secrets" section, set `GIST_ID` to the gistId of the gist you created in step 2.
+![](../../images/guides/gelato/task-creation-gist-id.png)
+
+** Note that the beta currently only supports triggering the task every minute. Once other time intervals are available,
+you will need to ensure that your current subscription plan with Gelato supports the desired number of monthly runs.
+See (Subscription Plans)[https://beta.app.gelato.network/subscriptions] **.
+
+#### 4. Fund the task
+
+There are two sets of fees that must be paid before the task can execute successfully
+
+**Gelato Fees**
+Gelato's fees are paid through [1Balance](https://docs.gelato.network/developer-services/1balance). Once you are whitelisted, visit the [one balance app](https://beta.app.gelato.network/balance)
+and deposit USDC on Polygon to top up your Gelato balance which will be used to pay the Gelato fees on all
+supported chains. The Testnet executions are subsidized by Gelato and free. Make sure to create your 1Balance account
+using the same wallet address that you used to create the Gelato task.
+
+Switch network to Polygon
+![](../../images/guides/gelato/switch-to-polygon.png)
+
+Deposit USDC
+![](../../images/guides/gelato/deposit-usdc.png)
+
+**Pyth Fees**
+These are the fees that will be paid out to the Pyth oracle. The Pyth fees are paid in the native token of the chain
```

Comment:
suggest replacing the first sentence with "Updating a pyth price feed requires the caller to pay a small fee". 

also, "Pyth fees" not "the pyth fees"